components/hubspot/actions/search-crm/search-crm.mjs

```
@@ -242,20 +268,30 @@ export default {
+    }
+
+    let results = await this.paginate({
       object: actualObjectType,
       data,
-      $,
     });
 
+    if (!exactMatch) {
+      results = results.filter((result) =>
+        result.properties[searchProperty]
+        && result.properties[searchProperty].toLowerCase().includes(searchValue.toLowerCase()));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)